utils/create_validate_pr_quickstarts.ts

```diff
@@ -125,13 +127,12 @@ export const createValidateQuickstarts = async (
   const quickstartErrors: string[] = [];
 
   logger.info(`Submitting ${quickstarts.length} quickstarts...`);
-  for (const c of chunk(quickstarts, 5)) {
```

Comment:
Since we're not using `chunk` here anymore, we may be able to remove the import at the top of the file.

Reply:
Done!